Start/start.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+LUNIServer.exe --world
```

Comment:
FWIW, if you put double quotes around the file name you won't have to rename it
`"LUNI Server.exe" --world`